src/v2/ion/RemediationConstants.js

```diff
@@ -131,4 +136,5 @@ export {
   AUTHENTICATOR_KEY,
   AUTHENTICATOR_METHODS,
   FORM_NAME_TO_OPERATION_MAP,
+  HINTS
```

Comment:
style-nit: Please add trailing comma (I think there is a lint rule for this too)